code/missionui/missiondebrief.cpp

```diff
@@ -1835,6 +1835,9 @@ void debrief_text_init()
 		}
 	}
 
+	// MageKing17 - Set the font so that wordwrapping in brief_color_text_init() calculates based on the same font as the debriefing itself.
```

Comment:
It's a little odd for you to add a comment saying it's from me without moving it from somewhere else.